site/_includes/explanations/non-persistent-topics.md

```diff
@@ -0,0 +1,57 @@
+As name suggests, non-persist topic does not persist messages into any durable storage disk unlike persistent topic where messages are durably persisted on multiple disks. 
```

Comment:
Since this is under `_includes` it won't create a page on its own. It can only be included from some other page.